ci-operator/step-registry/p2p/additional-labels/p2p-additional-labels-commands.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Login to cluster
+eval "$(cat "${SHARED_DIR}/api.login")"
+
+# Populate operator version labels
+cat "${SHARED_DIR}/operator-versions" >> "${SHARED_DIR}/firewatch-additional-labels"
```

Comment:
How about adding condition here to check if file `${SHARED_DIR}/operator-versions` exists? otherwise it would throw error in firewatch step I guess.

Reply:
Incase operator-install step didn't run and job failed before that.

Reply:
Agreed, fixed